app/controllers/backgrounds_controller.rb

```diff
@@ -2,13 +2,18 @@ class BackgroundsController < ApplicationController
   include BackgroundsHelper
   before_action :login_required
   before_action :no_demo_user
-  before_action :admin_required, except: [:create, :show_taxon_dist]
+  before_action :admin_required, except: [:create, :show_taxon_dist, :index]
```

Comment:
Why is admin required for index? 
This should not be the case since, and we will need to use it in future projects.

Reply:
This excludes index from being admin_required, if I understand your comment correctly.

Reply:
Oh yeah, sorry. Read it wrong.